product_docs/docs/pgd/5.7/rel_notes/pgd_5.7.0_rel_notes.mdx

```diff
@@ -61,6 +62,8 @@ guarantees that every transaction is decoded and sent at least once.</p>
 <table class="table w-100"><thead><tr><th>Component</th><th>Version</th><th>Description</th><th width="10%">Addresses</th></tr></thead><tbody>
 <tr><td>BDR</td><td>5.7.0</td><td>Fixed a server panic by ensuring that catalog lookups aren't performed in a callback that gets called very late in the commit cycle.</td><td></td></tr>
 <tr><td>BDR</td><td>5.7.0</td><td>Fixed an unintentional timed wait for a synchronous logical replication that resulted in unusually high latency for some transactions.</td><td>42273</td></tr>
+<tr><td>BDR</td><td>5.7.0</td><td><details><summary>The <code>bdr.group_camo_details</code> view now only lists data nodes belonging to the CAMO group.</summary><hr/><p>The <code>bdr.group_camo_details</code> view now only lists data nodes belonging to the CAMO commit scope group. Previously, the view could include logical standby nodes.</p>
+</details></td><td>45354</td></tr>
```

Comment:
Not only Logical Standby nodes. Previously, the view could include:

- Data nodes not belonging to the CAMO group;
- Logical Standby nodes;
- Subscriber-Only nodes;
- And even Witness nodes.